pyaerocom/aeroval/setupclasses.py

```diff
@@ -154,40 +160,31 @@ class StatisticsSetup(ConstrainedContainer):
 
     """
 
-    MIN_NUM = 1
-
-    def __init__(self, **kwargs):
-        self.weighted_stats = True
-        self.annual_stats_constrained = False
-        self.add_trends = False
-        self.trends_min_yrs = 7
-        self.stats_tseries_base_freq = None
-        self.use_fairmode = False
-        self.use_diurnal = False
-        self.obs_only_stats = False
-        self.model_only_stats = False
-        self.drop_stats = ()
-        self.stats_decimals = None
-
-        if "round_floats_precision" in kwargs:
-            precision = kwargs.pop("round_floats_precision")
-            set_float_serialization_precision(precision)
-
-        self.update(**kwargs)
-
-
-class TimeSetup(ConstrainedContainer):
-    DEFAULT_FREQS = ["monthly", "yearly"]
-    SEASONS = ["all", "DJF", "MAM", "JJA", "SON"]
-    main_freq = StrType()
-    freqs = ListOfStrings()
-    periods = ListOfStrings()
-
-    def __init__(self, **kwargs):
-        self.main_freq = self.DEFAULT_FREQS[0]
-        self.freqs = self.DEFAULT_FREQS
-        self.add_seasons = True
-        self.periods = []
+    MIN_NUM: PositiveInt = 1
+    weighted_stats: bool = True
+    annual_stats_constrained: bool = False
+    add_trends: bool = False
+    trends_min_yrs: PositiveInt = 7
+    stats_tseries_base_freq: str | None = None
```

Comment:
Why are some type annotations `X | None` (PEP 604, Python 3.10+) and others `Optional[X]`?
Does `pyndantic` supports PEP 604 style annotations?
If so, please use consistently.

Reply:
Yes. Will do.